### PR TITLE
exercises: Add a TOC

### PR DESCRIPTION
**Pull request by core-man**
Since we begin to work on seismological exercises, it's better to have an outline so that contributors may work on different sections. Based on the initial discussion in #205, some additional sections may be created:

- Introduction: **already created in the main branch**
- Earthquake catalog: **created in this PR**
- Metadata: **created in this PR**
- Waveform: **created in this PR**
- Other: **not added yet** (needs a better name and more discussion, any thought?)

Addresses #205.

### CODE REVIEW

File: source/exercises/metadata.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+元数据
```
Comment (seisman) [inline source/exercises/metadata.rst:1]: are you talking about stations here?

**Review by wangliang1989 — APPROVED** [review]
没有什么问题啊
但是，没有什么内容

**Comment by seisman** [comment]
> 没有什么问题啊
> 但是，没有什么内容

This is just an outline.

File: source/exercises/metadata.rst (hunk 1 above)
Comment (core-man) [reply source/exercises/metadata.rst:1]: Yes~

Comment (core-man) [reply source/exercises/metadata.rst:1]: Do we want another name, e.g., 台站信息？

**Comment by core-man** [comment]
> 没有什么问题啊
> 但是，没有什么内容

See #205 for what we might add to those different sections.

File: source/exercises/metadata.rst (hunk 1 above)
Comment (seisman) [reply source/exercises/metadata.rst:1]: Yes, metadata actually means more things.

Comment (core-man) [reply source/exercises/metadata.rst]: done in 9342191.

**Review by seisman — APPROVED** [review]

**Comment by core-man** [comment]
> * Other: **not added yet** (needs a better name and more discussion, any thought?)

How about **computation** or **calculation** (**计算方法**)? We may collect codes like `distance and angle`, `travel time curve`, `travel time path` here.

**Comment by seisman** [comment]
> > * Other: **not added yet** (needs a better name and more discussion, any thought?)
> 
> How about **computation** or **calculation** (**计算方法**)? We may collect codes like `distance and angle`, `travel time curve`, `travel time path` here.

They have low priority. We can discuss them later.

**Pull request merged**